src/server/graphql/mutations/helpers/promoteFirstTeamMember.js

```diff
@@ -0,0 +1,25 @@
+import publish from 'server/utils/publish';
+import {TEAM} from 'universal/utils/constants';
+import PromoteNewMeetingFacilitatorPayload from 'server/graphql/types/PromoteNewMeetingFacilitatorPayload';
+import getRethink from 'server/database/rethinkDriver';
+
+const promoteFirstTeamMember = (oldFacilitatorUserId, subOptions) => async (team) => {
+  const r = getRethink();
+  const now = new Date();
+  const {teamId, meetingId} = team;
+  await r.table('NewMeeting').get(meetingId).update((meeting) => ({
```

Comment:
This is a bit of a quibble, but I wonder if "NewMeeting" is a good name for this table. Versus something like "MeetingV2" or something versioned. I'd hate to have "NewNewMeeting" one day...

Reply:
agreed! it won't be forever. my thought is to refactor after the legacy meeting goes away.

Reply:
Word